org.mixedrealitytoolkit.tools/Tests/Editor/MRTK.Tools.EditorTests.asmdef

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "MixedReality.Toolkit.Tools.Editor.Tests",
+    "rootNamespace": "",
+    "references": [
+        "MixedReality.Toolkit.Core.TestUtilities",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
+    ],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.asset-store-validation",
+            "expression": "",
+            "define": "HAS_ASSET_STORE_VALIDATION"
+        },
+        {
+            "name": "org.mixedrealitytoolkit.core",
```

Comment:
Why would these packages ever _not_ have the Core package installed alongside, when they have the Core package as a dependency?

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/6bc4be65047cd3f65f608f2b6e406e1c31d72527/org.mixedrealitytoolkit.tools/package.json#L20

Reply:
Ahh....you caught me!

I copied and pasted across all packages for consistency.  Not all packages directly depend on "core" directly.

Seems like you have an opinion here, so I'll remove the conditions for packages that have Core as a direct dependency, but will leave the others.